src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2598,8 +2598,11 @@ public function get_autoloaded_options_size() {
 
 		$total_length = 0;
 
-		foreach ( $alloptions as $option_name => $option_value ) {
-			$total_length += strlen( $option_value );
+		foreach ( $alloptions as $option_value ) {
+			if ( is_array( $option_value ) || is_object( $option_value ) ) {
+				$option_value = serialize( $option_value ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.serialize_serialize
```

Comment:
The reason is that `serialize()` is what core uses to serialize options, so we need to use serialize to get the correct length, as it will be different length than what `wp_json_encode()` returns.

Reply:
Ok, that makes sense. In that case, we could use `maybe_serialize()`, which is what is used in `add_option()` and `update_option()` so we can avoid the need for a `phpcs:ignore`.

Reply:
Yeah, good idea.

Reply:
```suggestion
				$option_value = maybe_serialize( $option_value );
```